tests/test_playbooks/tasks/lifecycle_environment.yml

```
@@ -13,6 +13,7 @@
     name: "{{ lifecycle_environment_name }}"
     label: "{{ lifecycle_environment_label }}"
```

```suggestion
    label: "{{ lifecycle_environment_label | default(omit) }}"
```

the label is optional